docs/index.html

```diff
@@ -0,0 +1,13 @@
+<HTML>
```

Comment:
What do you think about placing this in deploy/* over docs/* (I know we had "docs" in the pages repo, but I guess time to fix that :) ?)

Reply:
Sure, I'll change the name of the folder.